manifests/install.pp

```diff
@@ -42,6 +42,49 @@
       path    => '/bin'
     }
   }
+
+  elsif $opendaylight::install_method == 'deb'{
+    
+    include apt
+    
+    apt::backports { 'jessie-backports':
+      comment  => 'This is Debian jessie-backports to install Java 8',
+      location => 'http://httpredir.debian.org/debian',
+      release  => 'jessie-backports',
+      repos    => 'main',
+      notify   => Exec['apt_update'],
```

Comment:
Umm...I haven't tested it completely. Basically this should add jessie-backports.
```
echo "deb http://httpredir.debian.org/debian jessie-backports main" > /etc/apt/sources.list.d/jessie-backports.list
apt-get update

```

That said, I don't think I have tested it properly. I have added Ubuntu 16.04 vagrant base-box in vagrant-opendaylight.

```
  # Box that installs ODL via Puppet deb method on Ubuntu 16.04
  config.vm.define "ubuntu16_pup_deb" do |ubuntu16_pup_deb|
    # Build Vagrant box based on Ubuntu 16.04
    ubuntu16_pup_deb.vm.box = "ubuntu/xenial64"

    # Install Puppet
    ubuntu16_pup_deb.vm.provision "shell", inline: "apt-get install -y puppet"
    ubuntu16_pup_deb.vm.provision "shell", inline: "apt-get install -y librarian-puppet"

    # Install OpenDaylight using its Puppet module
    ubuntu16_pup_deb.vm.provision "puppet" do |puppet|
      puppet.module_path = "modules"
      puppet.environment_path = "environments"
      puppet.environment = "main"
    end
  end
```
After that, I rsync my local puppet-odl changes: ``` vagrant-opendaylight$ rsync -r ~/puppet-opendaylight/* ./modules/opendaylight/``` and then  ``` vagrant up ubuntu16_pup_deb ```.

After that when executing ```puppet apply```, I get the following output:

```
[vagrant@localhost vagrant]$ puppet apply modules/opendaylight/manifests/install.pp 
Notice: Compiled catalog for localhost.iiit.ac.in in environment production in 0.24 seconds
Notice: Applied catalog in 0.01 seconds
```

I don't think this is expected behavior. Am I on the right track?